### DISCUSSION TITLE
Welcome to LightZero Discussions!

### DISCUSSION THREAD

**Discussion by puyuan1996** [opendilab/LightZero#176] [Announcements]
We’re using Discussions as a place to connect with other members of our community. We hope that you:

- Ask questions you’re wondering about.
- Share ideas.
- Engage with other community members.
- Welcome others and are open-minded. 
- Remember that this is a community we build together 💪.

To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ekiefl** [comment]
Hello, I'm Evan, and I'm the creator of [pooltool](https://github.com/ekiefl/pooltool), an open source billiards/pool simulation environment written in Python.

A few months ago I decided I would like to make a RL agent that can play pool. After some searching, I found LightZero. I like it because it seems modern and is actively developed. I'm glad I made this choice because my issues have never gone unnoticed, and everyone seems very open to helping.

It's nice to meet you all. Today I will make a post about the progress I've made on the project. I think collaborating with experts will be essential if the project is to succeed, so I would love if any of you are interested in getting involved.
